react-components/src/architecture/concrete/pointsOfInterest/fdm/PointsOfInterestFdmProvider.ts

```diff
@@ -26,8 +22,11 @@ export class PointsOfInterestFdmProvider implements PointsOfInterestProvider<Dms
     return await createPointsOfInterestInstances(this._fdmSdk, pois);
   }
 
-  async fetchAllPointsOfInterest(): Promise<Array<PointsOfInterestInstance<DmsUniqueIdentifier>>> {
-    return await fetchPointsOfInterest(this._fdmSdk);
+  async fetchPointsOfInterest(
+    _scene?: DmsUniqueIdentifier
+  ): Promise<Array<PointsOfInterestInstance<DmsUniqueIdentifier>>> {
+    throw Error('Not implemented:  Fdm PoI scene filter');
+    // return await fetchPointsOfInterest(this._fdmSdk);
```

Comment:
this returning is commented. Is it intentional?

Reply:
Yes, it's intentional.. Although it's a bit ugly. I was hoping we could have the FDM provider ready by Tuesday, but not sure anymore :/

Reply:
Hmmm ok. It's fine. We have to keep track on this to not let it live that long enough to forget that it still there.